version/version.go

```diff
@@ -67,7 +67,7 @@ func CheckMinimumVersion(versioner discovery.ServerVersionInterface) error {
 	// If no specific pre-release requirement is set, we default to "-0" to always allow
 	// pre-release versions of the same Major.Minor.Patch version.
```

Comment:
should we add a bit to the comments here, based on the code change?

Reply:
Sure, added the comment.